.circleci/config.yml

```diff
@@ -31,8 +31,10 @@ step_setup_global_packages: &step_setup_global_packages
       npm set cache ~/.npm
       sudo npm install -g npm@8.5.5
       npm ci
-      git submodule update --remote --init
+      git submodule update --init
       npm run provision:token:contracts
+      cd ./lib/safe-contracts/ && npm i || true && cd ../../
```

Comment:
This was because the post-install scripts they ran failed, but we needed the packages installed so we can run `hardhat`. Ended up going through `yarn` (which their repo uses anyway, so arguably should have started there) and skipping those scripts.